lib/travis/build/appliances/update_apt_keys.rb

```diff
@@ -0,0 +1,19 @@
+require 'travis/build/appliances/base'
+
+module Travis
+  module Build
+    module Appliances
+      class UpdateAptKeys < Base
+        def apply
+          command = <<-EOF
+          if command -v apt-get &>/dev/null && [[ -d /var/lib/apt/lists ]]; then
+            LANG=C apt-key list | awk -F'[ /]+' '/expired:/{printf "apt-key adv --recv-keys --keyserver keys.gnupg.net %s\\n", $3}' | sudo sh &>/dev/null
```

Comment:
Is it OK to assume that keys.gnupg.net has the new key?

Reply:
I believe it is a reasonable assumption that the GPG keys are on this pool.